proto/plans/query.proto

```diff
@@ -49,7 +49,7 @@ message QueryListResponse {
 
 message listInfoStruct {
 	string index = 1;
-	string name = 2;
```

Comment:
how do we know the name of the plan if we show only the description?

Reply:
this is from an old code (you probably didn't pull the latest commit). This was fixed in the new code